docs/self-managed/platform-deployment/helm-kubernetes/upgrade.md

```diff
@@ -89,6 +89,21 @@ For more details on the Keycloak upgrade path, you can also read the [Bitnami Ke
 
 ## Version update instructions
 
+### v8.2.3
+
+:::caution Breaking Change
+
+Zeebe Gateway authentication is now enabled by default. To authenticate,
+
+1. [Create a client credential](/docs/guides/setup-client-connection-credentials.md)
+2. [Assign permissions to the application](/docs/self-managed/identity/user-guide/authorizations/managing-resource-authorizations.md)
+3. Then input the client credentials to your client. For example:
+
+- For connecting to the Desktop Modeler, look [here](/docs/components/modeler/desktop-modeler/connect-to-camunda-cloud.md).
+- For connecting to zbctl, look [here](/docs/self-managed/zeebe-deployment/security/secure-client-communication/#zbctl)
```

Comment:
> For connecting to zbctl
zbctl is a tool used to connect to Zeebe, so the proposition here is probably "with" or something like that.

Reply:
You're right, it's definitely better english to use "with"! I also think I can improve the verb conjugation I used.